packages/bits/src/lib/toolbar/toolbar-keyboard.service.ts

```diff
@@ -2,6 +2,11 @@ import { Injectable } from "@angular/core";
 import { KEYBOARD_CODE } from "../../constants";
 import { MenuComponent } from "../menu";
 
+enum NavDirection {
+    LEFT = -1,
```

Comment:
Should this be `0`?

Reply:
I use it to get index for next element inside buttons array. Left - step back(-1) , current = 0, right - step forward(1);